docs/developer/kiosk.md

```diff
@@ -0,0 +1,106 @@
+# Kiosk
+
+To allow users to showcase their games in a fun and unique way, the games' Kiosk is added to Makecode Arcade. The purpose of the Kiosk is to further encourage users to upload their games. The Kiosk is an interface where a list of games is displayed in a carousel, letting a user scroll through what's currently highlighted to play. Hopefully, this will users excite users to want to upload their games.  
```

Comment:
I think the first sentence can leave off `games'` as Kiosk is more abstract, there is no true owner:
`the Kiosk is added to MakeCode Arcade`

Reply:
I think the second sentence can also be cut out to help avoid redundancy. 

```suggestion
To allow users to showcase their games in a fun and unique way, the games' Kiosk is added to Makecode Arcade. The Kiosk is an interface where a list of games is displayed in a carousel, letting a user scroll through what's currently highlighted to play. Hopefully, this will users excite users to want to upload their games.  
```